app/views/index.njk

```diff
@@ -2,10 +2,14 @@
 
 {% set primaryNavId = 'applications' %}
 
+{% set title %}
+  Applications ({{applications.length}})
+{% endset %}
+
 {% block content %}
 
   <h1 class="govuk-heading-l">
-    Applications
+    Applications ({{applications.length}})
```

Comment:
Use the `title` variable you set earlier, rather than repeating the string here.

Reply:
oh yeah